source/extensions/config_subscription/grpc/grpc_mux_failover.h

```diff
@@ -26,15 +24,30 @@ namespace Config {
  * Both the primary and failover streams are either SotW or Delta-xDS.
  *
  * The use of this class will be as follows: the GrpcMux object will own an instance of
- * the GrpcMuxFailover. The GrpcMuxFailover will own 2 GrpcStreams, primary and secondary.
+ * the GrpcMuxFailover. The GrpcMuxFailover will own 2 GrpcStreams, primary and failover.
  * Each of the primary and secondary streams will invoke GrpcStreamCallbacks
- * on their corresponding objects (also owned by the GrpcMuxFailoverProxy). These invocations
- * will be followed by the GrpcMuxFailoverProxy calling the GrpcStreamCallbacks on the GrpcMux
+ * on their corresponding objects (also owned by the GrpcMuxFailover). These invocations
+ * will be followed by the GrpcMuxFailover calling the GrpcStreamCallbacks on the GrpcMux
  * object that initialized it.
  *
- * Note: this class is WIP.
+ * Note: this class is WIP and should be considered alpha!
+ * At the moment, the supported policy is as follows:
+ * The GrpcMuxFailover attempts to establish a connection to the primary source. Once a response is
```

Comment:
will this documentation be eventually exposed to users? (when code is not alpha anymore)